msgcntr/messageforums-app/src/webapp/jsp/discussionForum/includes/dfAreaInclude.jsp

```diff
@@ -118,12 +118,13 @@ $(document).ready(function() {
                 <h:outputText  value=" | " rendered="#{ForumTool.newForum || ForumTool.instructor || forum.changeSettings}" />
 
                     <%-- link to display other options on this forum --%>
-                    <f:verbatim><a href="#" class="moreMenuLink"></f:verbatim>
+                    <f:verbatim><a href="#" class="moreMenuLink" aria-controls="expandable-content" aria-expanded="false"></f:verbatim>
```

Comment:
Does aria-expanded ever get set to true?

Reply:
@i-a-m-s-k please let us know if you need help here .... this is a good start ...

Reply:
Thank you for the suggestion, I will try to incorporate the changes (data-bs-toggle) and check if it works.